NebulaModel/DataStructures/MechaData.cs

```diff
@@ -97,6 +106,29 @@ public void Deserialize(INetDataReader reader)
         ReactorStorage.Import(br);
         WarpStorage.Import(br);
         Forge.Import(br);
+        ConstructionModule.Import(br);
+    }
+
+    public void UpdateMech(Player destination)
+    {
+        destination.package = Inventory;
+        using (var ms = new MemoryStream())
+        {
+            var bw = new BinaryWriter(ms);
+            DeliveryPackage.Export(bw);
+            ms.Seek(0, SeekOrigin.Begin);
+            var br = new BinaryReader(ms);
+            destination.deliveryPackage.Import(br);
+            DeliveryPackage = destination.deliveryPackage;
+        }
```

Comment:
Would it make sense to abstract these low-level portions into a separate class function?

Reply:
yeah, I guess it can be separated as a function to handle binary export/import for other classes